graspy/align/sign_flips.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+
+from .base import BaseAlign
+
+
+class SignFlips(BaseAlign):
+    """
+    This module is used to transform two datasets by flipping the signs of all
+    entries along some dimensions. In particular, it does so in a way that
+    brings these two datasets to the same orthant according to some criterion,
+    computed along each dimension. The two critera currently available are the
+    median and the maximum value in magnitude along each dimension.
+
+    There are two choices for which orthant the datasets end up in: either the
+    first orthant (i.e. with all criteras being positive), which is a default,
+    or and orthant in which one the second dataset is already in.
```

Comment:
```suggestion
```

Reply:
reason being this part is super hard to explain, and I think it may actually be unnecessary here. feel free to ignore, but if you do, wording could use clarification

Reply:
also is it no longer true that we support both of these?

Reply:
you are correct. we no longer support both, the comment is archaic.

(but you can always pass identity as Y if just want to first-orthantize X :) this isn't specified anywhere though. i thought that making I a default only in this sepcific class would introduce confusion.

Reply:
i probably should rewrite that whole doc strong

Reply:
done. also added a throwaway comment regarding passing identity as Y. something that can be explained in more details in the tutorial.